### PR TITLE
Weather Layout: html, gray background and 2 temporary icons

### PR DESCRIPTION
**Pull request by CooKingBeast**
<Label> not selfclosed because of Webstorm warining that this solution doesn't work in all browsers

### CODE REVIEW

File: Weather/index.html [hunk 1]
```diff
@@ -6,7 +6,40 @@
 		<link rel="stylesheet" href="./style.css" />
 	</head>
 	<body>
-		<h1>Title test</h1>
+		<div class="wrapper">
+			<main class="main_info">
```
Comment (Noch4nce) [inline Weather/index.html]: use the structure HTML5 everywhere or nowhere

File: Weather/index.html [hunk 2]
```diff
@@ -6,7 +6,40 @@
...
+				<span class="deegrees">16&#8451</span>
+				<span class="city">London</span>
+				<div class="weather_type">
+					<img src="./assets/cloudyIMG.png" alt="cloudyIMG" />
+					<span>Cloudy</span>
+				</div>
+			</main>
+			<div class="content">
+				<div class="form">
```
Comment (Noch4nce) [inline Weather/index.html]: change to a special element

File: Weather/index.html [hunk 3]
```diff
@@ -6,7 +6,40 @@
...
+					<label for="form_input">
+					<input id="form_input" type="text" />
```
Comment (Noch4nce) [inline Weather/index.html]: placeholder ?

File: Weather/style.css [hunk 4]
```diff
@@ -6,3 +6,8 @@ body {
 	margin: 0;
 	padding: 0;
 }
+
+.wrapper {
+	background-color: gray;
+	height: 100vh;
```
Comment (Noch4nce) [inline Weather/style.css]: mayby global ?

**Review by Noch4nce — APPROVED** [review]

File: Weather/style.css (hunk 4 above)
Comment (CooKingBeast) [reply Weather/style.css]: ок

**Pull request merged**